packages/editor/src/components/post-author/index.js

```diff
@@ -1,49 +1,165 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __, _n } from '@wordpress/i18n';
 import { withInstanceId, compose } from '@wordpress/compose';
 import { Component } from '@wordpress/element';
 import { withSelect, withDispatch } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
 
 /**
  * Internal dependencies
  */
 import PostAuthorCheck from './check';
 
+/**
+ * External dependencies
+ */
+import Autocomplete from 'accessible-autocomplete/react';
+
```

Comment:
Unneeded newline should be removed to keep dependencies grouping intact.

```suggestion
```

Reply:
already addressed in reordering.